liquibase/src/main/resources/changelog/0001-create-system-table.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="051a0026-c808-47be-90f9-3fa6a8d6e246" author="daniel-j-mason ">
+        <createTable tableName="system">
+            <column name="id" type="uuid">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="name" type="varchar(255)">
+                <constraints nullable="false" unique="true"/>
+            </column>
+            <column name="elite_id" type="bigint"/>
+            <column name="x_coordinate" type="FLOAT8"/>
```

Comment:
It is better to use `float` because it is a base supported liquibase type, and will make the changeset system agnoistic